src/mono/netcore/sample/wasm/WasmSample.csproj

```diff
@@ -39,11 +43,24 @@
       MicrosoftNetCoreAppRuntimePackDir="$(MicrosoftNetCoreAppRuntimePackDir)"
       MainAssembly="bin\WasmSample.dll"
       MainJS="$(MonoProjectRoot)wasm\runtime-test.js"
-      AssemblySearchPaths="@(AssemblySearchPaths)"/>
+      AssemblySearchPaths="@(AssemblySearchPaths)"
+      FilesToIncludeInFileSystem="@(WasmFilesToIncludeInFileSystem)"
+   />
     <Exec Command="chmod a+x $(AppDir)/run-v8.sh" />
   </Target>
 
+  
+
+  <ItemGroup>
+    <PackageReference Include="xunit" Version="2.4.0" />
```

Comment:
Why do we need a reference to `xunit`?

Reply:
This was for writing tests in the Wasm sample app.